modules/EnsEMBL/Web/Component/Variation/Explore.pm

```diff
@@ -97,6 +101,10 @@ sub content {
     {'title' => 'Citations',                                          'img' => '96/var_citations.png',              'url' => $cit_url,      'count' => $cit_count   },
   );
 
+  if ($self->hub->species eq 'Homo_sapiens') {
```

Comment:
I think for now this is fine but what if other species have this enable? Or will it just be human? One way of doing it is having a flag in each species ini file, this way to enabling it for another species will just be adding the flag in the INI file, its not critical for now if it is going to be only human but something to remember for the future